packages/block-library/src/breadcrumbs/index.php

```diff
@@ -17,23 +17,34 @@
  * @return string Returns the post breadcrumb for hierarchical post types.
  */
 function render_block_core_breadcrumbs( $attributes, $content, $block ) {
-	// Exclude breadcrumbs from special contexts.
-	if ( is_home() || is_front_page() ) {
+	$is_home_or_front_page = is_home() || is_front_page();
+	if ( ! $attributes['showOnHomePage'] && $is_home_or_front_page ) {
 		return '';
 	}
 
 	$breadcrumb_items = array();
 
 	if ( $attributes['showHomeLink'] ) {
-		$breadcrumb_items[] = sprintf(
-			'<a href="%s">%s</a>',
-			esc_url( home_url() ),
-			esc_html__( 'Home' )
-		);
+		// On home/front page, show as current page instead of link.
+		if ( $is_home_or_front_page ) {
+			$breadcrumb_items[] = sprintf(
+				'<span aria-current="page">%s</span>',
+				esc_html__( 'Home' )
+			);
+		} else {
+			$breadcrumb_items[] = sprintf(
+				'<a href="%s">%s</a>',
+				esc_url( home_url() ),
```

Comment:
We'll need to add the `/` as an argument to prevent the extra redirect that will be caused by navigating to the URL without a trailing slash:

<img width="628" height="45" alt="Image" src="https://github.com/user-attachments/assets/3a598fc8-051f-48d6-aa2c-ff505805e8bc" />

```suggestion
				esc_url( home_url( '/' ) ),
```

Reply:
TIL, thanks!